### PR TITLE
Add queries to config.yml

### PR DESCRIPTION
**Pull request by stephenpmurray**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -19,6 +19,47 @@ domains:
       - name: Block charts
         description: Fetch the latest versions of all charts in the marketplace
         query: query.blockCharts
+      - name: Publish Block Chart
+        description: Publish a new Block Chart based on an existing template
+        query: mutation.publishBlockChart
+      - name: Delete Block Chart
+        description: Delete a new Block Chart based on an existing template
```
Comment (fenech) [inline config.yml:26]: I don't think "new" applies here

File: config.yml [hunk 2]
```diff
@@ -19,6 +19,47 @@ domains:
...
+        description: Fetch the default config of a Crossplane Chart
+        query: query.crossplaneChart
+      - name: Crossplane Charts
+        description: Fetch a list of the latest versions of all Crossplane Charts
+        query: query.crossplaneCharts
+      - name: Template Chart
+        description: Fetch the default config of a Crossplane Chart
+        query: query.templateChart
+      - name: Template Charts
+        description: Fetch the latest versions of all Templates
```
Comment (fenech) [inline config.yml:44]: How about "Publisher Templates" or something like that?

File: config.yml [hunk 3]
```diff
@@ -32,6 +73,78 @@ domains:
       - name: Remove device
         description: Unregister a device
         query: mutation.removeDevice
+      - name: IPMI Command
+        description: Run an IPMI command against a device (deprecated)
+        query: mutation.ipmiCommand
```
Comment (fenech) [inline config.yml:78]: Let's remove this totally dead endpoint from the code rather than documenting it here!

File: config.yml [hunk 4]
```diff
@@ -32,6 +73,78 @@ domains:
...
+        description: Fetch a single Federation Host
+        query: query.federationHost
+      - name: Edit Federation
+        description: Edit a federation
+        query: mutation.editFederation
+      - name: Create Federation Guest
+        description: Create a Federation Guest User
+        query: mutation.createFederationGuest
+      - name: Edit Federation Guest
+        description: Edit an existing Federation Guest User
```
Comment (fenech) [inline config.yml:102]: I don't think the word "user" is right, as it implies a human. Maybe "account" instead?

File: config.yml [hunk 5]
```diff
@@ -32,6 +73,78 @@ domains:
...
+        query: mutation.createFederationHost
+      - name: Edit Federation Host
+        description: Edit an existing Federation Host
+        query: mutation.editFederationHost
+      - name: Remove Federation Host
+        description: Remove an existing Federation Host
+        query: mutation.removeFederationHost
+
+  - name: Management
+    description: Endpoints for managing your installtion
```
Comment (fenech) [inline config.yml:121]: typo

File: config.yml [hunk 6]
```diff
@@ -32,6 +73,78 @@ domains:
...
+        query: query.oktoResourceLogs
+
+  - name: Provisions
+    description:  View, create and manage provisions
+    usecases:
+      - name: Provisions
+        description: List all Provisions in a site
+        query: query.provisions
+      - name: Provision
+        description: Create a provision and return a device
```
Comment (fenech) [inline config.yml:144]: The return type will be shown in the docs, so no need to include it here

File: config.yml [hunk 7]
```diff
@@ -83,22 +208,31 @@ domains:
...
+        description: Remove a site
         query: mutation.removeSite
+      - name: Create org
+        description: Create a new org
+        query: mutation.createOrg
+      - name: Edit org
+        description: Modify an existing org
+        query: mutation.editOrg
+      - name: Remove org
+        description: Remove a org
```
Comment (fenech) [inline config.yml:227]: a horg

File: env/docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+      - APIGW_JWKS_URL=""
+      - APIGW_VALIDATE_JWT=false
+      - CONNARTIST_PORT=""
+      - DEVICEMAN_GRPC_ADDR=""
+      - SERVICEMAN_GRPC_ADDR=""
+      - SETTINGS_GRPC_ADDR=""
+
+  api-docs:
+    image: apidocs
+    build:
```
Comment (fenech) [inline env/docker-compose.yml:19]: The short form `build: .` would do it

File: env/docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+      context: ./
+    depends_on:
+      - api-gw
+    volumes:
+      - ../public:/public:Z
```
Comment (fenech) [inline env/docker-compose.yml:24]: lowercase `:z` would be fine, no need for exclusive access here

File: config.yml (hunk 3 above)
Comment (stephenpmurray) [reply config.yml:78]: https://github.com/nbycomp/graphql-hello-world/pull/1023

File: env/docker-compose.yml (hunk 8 above)
Comment (stephenpmurray) [reply env/docker-compose.yml:19]: I'd actually messed it up. changed it to `../`

**Pull request merged**